documentation/docs/06-hooks.md

```diff
@@ -67,15 +67,15 @@ You can add call multiple `handle` functions with [the `sequence` helper functio
 `resolve` also supports a second, optional parameter that gives you more control over how the response will be rendered. That parameter is an object that can have the following fields:
 
 - `ssr: boolean` (default `true`) — if `false`, renders an empty 'shell' page instead of server-side rendering
-- `transformPage(opts: { html: string }): string` — applies custom transforms to HTML
+- `transformPageChunk(opts: { html: string, done: boolean }): string` — applies custom transforms to HTML. If `done` is true, it's the final chunk
```

Comment:
I don't know if there's anything we should say here, but I'm not that familiar with how the chunking happens. Does it match a single write to the stream? Or is it split up so that each chunk is less than the TCP window size or something? E.g. I'm wondering if there's any risk that the `.replace('old', 'new')` example doesn't work because "ol" is one chunk and "d" in the next chunk

Reply:
I addressed that above:

> Chunks are not guaranteed to be well-formed HTML (they might include the opening tag of an element but not its closing tag, for example) but nor are they arbitrary — they will always be split at sensible boundaries (like `%sveltekit.[property]%` or before the contents of a layout or page component)

I figured there wasn't much point going into detail in the docs until we actually start streaming responses

Reply:
I guess I should read PR descriptions :smile: Still, it might be worth including here so that people write their code correctly from the beginning and so that we don't break their applications when we do start streaming

Reply:
added some detail